pkgs/development/python-modules/requests/0001-Prefer-NixOS-Nix-default-CA-bundles-over-certifi.patch

```diff
@@ -0,0 +1,59 @@
+From 4d5a0213dc649bc5cfd047c0b5c672614d67d4c9 Mon Sep 17 00:00:00 2001
+From: Keshav Kini <keshav.kini@gmail.com>
+Date: Sun, 16 May 2021 20:35:24 -0700
+Subject: [PATCH] Prefer NixOS/Nix default CA bundles over certifi
+
+Normally, requests gets its default CA bundle from the certifi
+package.  On NixOS and when using Nix on non-NixOS platforms, we would
+rather default to using our own certificate bundles controlled by the
+Nix/NixOS user.
+
+This commit overrides certifi's where() function to do the following:
```

Comment:
I clarified that sentence as follows:

```
This commit overrides requests.certs.where(), which previously was
just aliased to certifi.where(), so that now it does the following:
```